pendulum_control/include/pendulum_control/rtt_kpsr_executor.hpp

```diff
@@ -24,32 +10,31 @@
 #include "rttest/rttest.h"
 #include "rttest/utils.hpp"
 
-#include "rmw/rmw.h"
-
-#include "rclcpp/executor.hpp"
-#include "rclcpp/macros.hpp"
-#include "rclcpp/memory_strategies.hpp"
+#include <kpsr_ros2_executor/executor_factory.hpp>
+#include <kpsr_ros2_executor/streaming_executor.hpp>
 
 namespace pendulum_control
 {
-/// Instrumented executor that syncs Executor::spin functions with rttest_spin.
-class RttExecutor : public rclcpp::Executor
```

Comment:
I think we can keep the name and the inheritance from the `Executor` interface

Reply:
Done